src/Core/Service/ServiceClient.php

```diff
@@ -19,24 +17,10 @@ class ServiceClient
     public function __construct(
         public readonly HttpClientInterface $client,
         private readonly string $shopwareVersion,
-        private readonly ServiceEntry $entry,
-        private readonly Filesystem $filesystem,
+        private readonly ServiceEntry $entry
     ) {
     }
 
-    public function downloadAppZipForVersion(string $zipUrl, string $destination): AppInfo
-    {
-        $response = $this->client->request('GET', $zipUrl, [
-            'headers' => [
-                'Accept' => 'application/zip',
-            ],
-        ]);
-
-        $this->checkResponse($response);
-
-        return $this->parseAppZipResponse($response, $destination);
-    }
-
     public function latestAppInfo(): AppInfo
```

Comment:
This info also comes from the registry? Should we refactor the code using this to also use the service client?

Reply:
yes, but out of context here. I'm just removing stuff directly related to the PR. This should be done later.